Devscord.DiscordFramework/WorkflowBuilder.cs

```diff
@@ -90,6 +111,18 @@ private void AddHandlers<T>(Action<WorkflowBuilderHandlers<T>> action, Action<T>
             }
         }
 
+        public WorkflowBuilder AppendOnUserAddedReaction(Action<ReactionContext> action)
```

Comment:
Append zmień na Add - wszędzie tutaj mamy nazwe od Add

Reply:
Dałem Append bo chciałem trochę to odróżnić od metod, które dodają handlery. Czy na pewno mam to zmienić?

Reply:
i też konfiguracja z samym odpalaniem reakcji powinna być wbudowana w framework 

WorkflowBuilder jest po to, żeby użytkownicy frameworka mogli dorzucić coś własnego
Np dodatkową niestandardową obsługe wiadomości
Albo dodatkową obsługe witania ludzi 
Albo dodatkową logike do odpalania bota 

Traktuj framework i bota jako osobne aplikacje
framework powinien być zrobiony tak
żebyśmy mogli swobodnie wykorzystać go tworząc innego bota 

swobodnie, czyli wszystko powinno działać jak najszybciej, nie powinniśmy wtedy musieć myśleć o dziwnych zależnościach 

oczywiście - takie zależności sie zdarzają, np wymuszenie korzystania z autofaca 
no ale to byłoby akurat proste to podmiany, więc to nie jest jakiś duży problem